instrumentation/netty/netty-3.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v3_8/server/NettyNetServerAttributesGetter.java

```diff
@@ -14,23 +14,32 @@
 import java.net.SocketAddress;
 import javax.annotation.Nullable;
 import org.jboss.netty.channel.socket.DatagramChannel;
+import org.jboss.netty.handler.codec.http.HttpResponse;
 import org.jboss.netty.handler.codec.http.HttpVersion;
 
 final class NettyNetServerAttributesGetter
-    implements NetServerAttributesGetter<HttpRequestAndChannel> {
+    implements NetServerAttributesGetter<HttpRequestAndChannel, HttpResponse> {
 
   @Override
   public String getTransport(HttpRequestAndChannel requestAndChannel) {
     return requestAndChannel.channel() instanceof DatagramChannel ? IP_UDP : IP_TCP;
   }
 
   @Override
-  public String getProtocolName(HttpRequestAndChannel requestAndChannel) {
+  public String getNetworkTransport(
+      HttpRequestAndChannel requestAndChannel, HttpResponse response) {
+    return requestAndChannel.channel() instanceof DatagramChannel ? "udp" : "tcp";
```

Comment:
nit: This exists in at least 7 places. Might be nice to encapsulate in a static util method.